internal/server/v0/placestatvar/golden/get_place_stat_vars/california.json

```diff
@@ -8022,46 +8022,6 @@
         "Count_Person_18To64Years_PovertyStatusDetermined",
         "Count_Person_18To64Years_PovertyStatusDetermined_ResidesInGroupQuarters",
         "Count_Person_18To64Years_PovertyStatusDetermined_ResidesInNoninstitutionalizedGroupQuarters",
-        "Count_Person_18To64Years_RatioToPovertyLine1.38To4",
-        "Count_Person_18To64Years_RatioToPovertyLine1.38To4_BlackOrAfricanAmericanAlone",
-        "Count_Person_18To64Years_RatioToPovertyLine1.38To4_Female",
```

Comment:
curious why these all got deleted?

Reply:
The deletions are removed after the SAHIE import update.